crates/jrsonnet-evaluator/src/arr/mod.rs

```diff
@@ -100,28 +103,29 @@ impl ArrValue {
 		Self::new(RangeArray::new_inclusive(a, b))
 	}
 
+	/// # Panics
+	/// If step == 0
 	#[must_use]
-	pub fn slice(
```

Comment:
ArrValue::slice was refactored to maintain same behavior as `std.slice`, it is now supports negative indices and other things